### PR TITLE
falkon: add KDE build option.

### PR DESCRIPTION
**Pull request by travankor**

### CODE REVIEW

**Comment by pullmoll** [comment]
Falkon is used also as the default browser for lxqt. I believe this change may seriously increase the memory footprint for lxqt and the live images. This is bad because lxqt is usually a choice for old machines with less memory and disk space.
IMHO the `build_options_default` should be empty. The default browser for KDE is `konqueror`, or is it?

**Comment by pullmoll** [comment]
As an alternative I think, since building `falkon` is not too heavy, i.e. doesn't take many, many hours like chromium, wbkit2gtk, or qt5-webengine (which is what it uses), we could create two versions: `falkon` and `falkon-kde` the latter built with the additional options.

**Comment by travankor** [comment]
>I believe this change may seriously increase the memory footprint for lxqt and the live images.

Do you mean building with the optional KDE integration libraries? Because that's already what's happening in the current build, so I just set the existing behaviour as the default. If not, could you explain more what you mean?

The motivation for this PR was to create a more stripped-down version of falkon that did not require optional KDE libraries.

**Comment by pullmoll** [comment]
Uh, okay, I got that wrong. I thought you were adding new dependencies with this PR.
I'll leave this to other @void-linux/pkg-committers then.

File: srcpkgs/falkon/template
Comment (Piraty) [inline srcpkgs/falkon/template:22]: put `_default` below `desc`

**Review by Piraty — CHANGES_REQUESTED** [review]
could you please
1. comment the motivation (which i second) in the commit
2. think about using `vopt_if()` idiom

File: srcpkgs/falkon/template
Comment (travankor) [reply srcpkgs/falkon/template]: thanks, i didn't think to have multiple values in `vopt_if`

**Comment by ericonr** [comment]
@Piraty ok to merge?

**Pull request merged**
